src/runtime/offers-flow-test.js

```diff
@@ -99,6 +100,41 @@ describes.realWin('OffersFlow', {}, (env) => {
     await offersFlow.start();
   });
 
+  it('should have valid OffersFlow constructed, routed to the new offers iframe', async () => {
+    sandbox
+      .stub(runtime.clientConfigManager(), 'getClientConfig')
+      .callsFake(() => {
+        return Promise.resolve(
```

Comment:
Could `.callsFake(() => { return Promise.resolve(` be replaced with `.resolves(` ?

Reply:
Ditto.